config/nas_mirror.sh

```
@@ -0,0 +1,70 @@
+    | sed "s|^$SOURCE/||" | sort
+)
+
+mapfile -t DEST_SNAPS < <(
+    find "$DEST" -mindepth 3 -maxdepth 3 -type d \
+    | sed "s|^$DEST/||" | sort
+)
+
+# Convert DEST snapshots to lookup table
+declare -A DEST_SET
+for s in "${DEST_SNAPS[@]}"; do
+    DEST_SET["$s"]=1
```

Shellcheck (0.11.0)

[warning] DEST_SET appears unused. Verify use (or export if used externally).

(SC2034)